tests/drivers/build_all/src/Makefile

```diff
@@ -0,0 +1,3 @@
+ccflags-y += -I${ZEPHYR_BASE}/tests/include
```

Comment:
I believe there're no Makefiles in Zephyr anymore (they're generated with CMake).

Reply:
Thanks for you comment! 

Without this change, './scripts/sanitycheck --platform qemu_cortex_m3 --enable-slow -b'  will hit 'tc_util.h: no such file or directory'

Reply:
Good, but you would need to re-do it using CMakeLists.txt instead.